apps/server/src/modules/provisioning/interface/provider.response.interface.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface IProviderResponse {}
```

Comment:
Is the use of bare generic sufficient here? IProviderResponse does not contain anything, and could be removed then.

Reply:
At the moment we dont really need it, but we expect that the responses have general properties which could be placed here.